src/views/Register/components/ValidationRules.vue

```diff
@@ -1,70 +1,104 @@
 <template>
-  <el-card>
-    <h2>Register</h2>
-    <el-form
-      class="register-form"
-      :model="model"
-      :rules="rules"
-      ref="form"
-      @submit.native.prevent="Register"
-    >
-      <el-form-item prop="username">
-        <el-input v-model="model.username" placeholder="Username"></el-input>
-      </el-form-item>
+    <el-card>
+        <h2>Register</h2>
+        <el-form
+                class="register-form"
+                :model="model"
+                :rules="rules"
+                ref="form"
+                @submit.native.prevent="register"
+        >
+            <el-form-item prop="username">
+                <el-input v-model="model.username" placeholder="Username"></el-input>
+            </el-form-item>
 
-      <el-form-item prop="email">
-        <el-input v-model="model.email" placeholder="Email" type="email">
-        </el-input>
-      </el-form-item>
+            <el-form-item prop="email">
+                <el-input v-model="model.email" placeholder="Email" type="email">
+                </el-input>
+            </el-form-item>
 
-      <el-form-item prop="password">
-        <el-input
-          v-model="model.password"
-          placeholder="Password"
-          type="password"
-        ></el-input>
-      </el-form-item>
+            <el-form-item prop="password">
+                <el-input
+                        v-model="model.password"
+                        placeholder="Password"
+                        type="password"
+                ></el-input>
+            </el-form-item>
 
-      <el-form-item prop="confirmPassword">
-        <el-input
-          v-model="model.confirmPassword"
-          placeholder="Confirm Password"
-          type="password"
-        ></el-input>
-      </el-form-item>
+            <el-form-item prop="confirmPassword">
+                <el-input
+                        v-model="model.confirmPassword"
+                        placeholder="Confirm Password"
+                        type="password"
+                ></el-input>
+            </el-form-item>
 
-      <el-form-item>
-        <el-button
-          class="register-button"
-          type="primary"
-          native-type="submit"
-          block
-          >Register now</el-button
-        >
-      </el-form-item>
-    </el-form>
-  </el-card>
+            <el-alert type="error" v-model="errors" v-for="error in errors" :key="error[0]">
+                <li>{{error[0]}}</li>
+                <li>{{error[1]}}</li>
+            </el-alert>
```

Comment:
hmmm... there is no cleaner way to get and list all errors instead of `[0]` and `[1]`?